surveys/notes/process.md

```diff
@@ -0,0 +1,31 @@
+# Process
+
+Proposed timeline:
```

Comment:
This timeline is already included here: https://github.com/rust-lang/surveys/tree/main/surveys/2021-annual-survey#tentative-timeline

Not sure we need both.